.circleci/docker/common/install_conda.sh

```diff
@@ -97,17 +97,19 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   # Pin SciPy because of failing distribution tests (see #60347)
   # Pin MyPy version because new errors are likely to appear with each release
   # Pin hypothesis to avoid flakiness: https://github.com/pytorch/pytorch/issues/31136
-  as_jenkins pip install --progress-bar off pytest \
-    scipy==1.6.3 \
-    scikit-image \
-    psutil \
-    unittest-xml-reporting \
-    boto3==1.16.34 \
-    hypothesis==4.53.2 \
-    expecttest==0.1.3 \
-    mypy==0.812 \
-    tb-nightly \
-    librosa>=0.6.2
+  # as_jenkins pip install --progress-bar off boto3==1.16.34 \
```

Comment:
I'm guessing you will remove this commented out part? ;)

Reply:
Yes, i just removed it